tests/robot-tests/tests/libs/admin-common.robot

```diff
@@ -29,7 +29,7 @@ user signs in as analyst1
 
     user waits until h1 is visible  Dashboard
     user waits until page contains title caption  Welcome Analyst1
-    user waits until page contains element   id:publicationsReleases-themeTopic-themeId   180
+    user waits until page contains element   css:#publicationsReleases-themeTopic-themeId,[data-testid='no-permission-to-access-releases']   180
```

Comment:
Don't think the comma here is necessary?

Reply:
I think this is necessary. The selector is either looking for either `#publicationsReleases-themeTopic-themeId` or `[data-testid='no-permission-to-access-releases']`. The first appears if analyst1 does have permission view/edit releases - the second appears is analyst1 has no permission to view/edit any releases. This covers both cases.

Reply:
ah that's pretty cool. Didn't realise that was possible